index.html

```diff
@@ -79,6 +79,42 @@ <h3 class="subheading center">Quality Ecosystem</h3>
     </div>
   </div>
 
+  <div class="whatisgulp">
+    <div class="container-sml">
```

Comment:
Larger container - theres too much empty space around the editor so it looks a little funky